src/app/daemon_linux.cc

```diff
@@ -90,14 +90,14 @@ bool DaemonLinux::Start() {
     chdir(d.c_str());
     // close stdin/stdout/stderr descriptors
     ::close(0);
-    if(::open("/dev/null", O_RDWR)<0)
-	    exit(1);
+    if (::open("/dev/null", O_RDWR) < 0 )
+      exit(1);
```

Comment:
Thanks @guzzijones. We should probably return `false` instead of `exit(1)` so our daemon instance can cleanup with `Stop()` (see `main.cc`)